data/expected/padmm_serial/test_parameters

```diff
@@ -0,0 +1,18 @@
+//used wavelets
```

Comment:
Do we need files like this for the mpi tests as well?

Reply:
It is probably a good idea!